ci/pipelines/deploy-to-production.yml

```diff
@@ -346,6 +356,60 @@ groups:
       - update-deploy-to-prod-pipeline
 
 jobs:
+  - name: deploy-carbon-relay-to-prod
```

Comment:
I think positioning it first in the `jobs` list affects the ordering of the tabs in the pipeline - we should prob keep it consistent with staging and test and stick it between toolbox and the sidecars.

Reply:
Fixed